third_party/docfx-doclet-143274/src/test/java/com/microsoft/util/YamlUtilTest.java

```diff
@@ -127,7 +127,12 @@ public void cleanupHtmlEqualTitlesTest() {
         assertEquals(expectedResult, YamlUtil.cleanupHtml(expectedActual));
         assertEquals(random + expectedResult + random, YamlUtil.cleanupHtml(random + expectedActual + random));
         assertEquals(expectedResult + random + expectedResult, YamlUtil.cleanupHtml(expectedActual + random + expectedActual));
+
         assertEquals("= text =", YamlUtil.cleanupHtml("= text ="));
+        assertEquals("==testing==", YamlUtil.cleanupHtml("==testing=="));
```

Comment:
I would put `"scikit-learn":"==0.19.0"` thing as a test case.

Reply:
Ah yes! I'll add that